test/libsolidity/ASTJSON/not_existing_import.sol

```diff
@@ -6,4 +6,3 @@ contract C is NotExisting.X
 }
 
 // ----
-// failAfter: Parsed
```

Comment:
Any idea about why this was removed? And also the other test that removed the same `// failAfter: Parsed` line.

Reply:
More like here: https://github.com/ethereum/solidity/pull/10996#discussion_r1180495429

Basically, this change should not be in the PR. Isoltest has a bug: #14093.

Reply:
Yup, figures as much, but had no idea we had an issue for that. Thanks @cameel!

Reply:
> More like here: [#10996 (comment)](https://github.com/ethereum/solidity/pull/10996#discussion_r1180495429)
> 
> Basically, this change should not be in the PR. Isoltest has a bug: #14093.

So this means that @nikola-matic should undo these changes?

Reply:
Yeah, I've undone them already.

Reply:
So why the change still here?

Reply:
Huh, weird.